release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -152,7 +152,7 @@ Ingress Node Firewall Operator was made a technology preview in {product-title}
 ==== Dynamic use of non-reserved CPUs for OVS
 
 With this release, the Open vSwitch (OVS) networking stack can dynamically use non-reserved CPUs.
-This dynamic use of non-reserved CPUs occurs by default in performance-tuned clusters with a CPU manager policy set to `static`.
+This dynamic use of non-reserved CPUs occurs by default in nodes in a `MachineConfigPool` with a performance profile applied.
```

Comment:
```suggestion
This dynamic use of non-reserved CPUs occurs by default in nodes in a `MachineConfigPool` with a `PerformanceProfile` applied.
```

Just to maintain the style when referring to objects.

Reply:
Updated accordingly @MarSik